### PR TITLE
Add logo for BLxPR

### PR DESCRIPTION
**Pull request by Primedao-admin**
#### Hi! What is your PR about?

- [x] Add images for your space
- [x] Add custom domain for your space
- [ ] Add alias for migrate your space
- [ ] Add or edit a skin
- ~Edit your space with GitHub?~ (deprecated) Please migrate your space to ENS https://docs.snapshot.page/spaces/migrate
- ~Create your space with GitHub~ (deprecated) Create your space with ENS instead https://docs.snapshot.page/spaces/create

### CODE REVIEW

File: spaces/blxpr.eth/blxpr.eth [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (gokaiorg) [inline spaces/blxpr.eth/blxpr.eth:1]: You will need to delete this file.

**Review by gokaiorg — CHANGES_REQUESTED** [review]
Rename your images, they have a `.png.png` extension.

File: spaces/blxpr.eth/blxpr.eth (hunk 1 above)
Comment (Primedao-admin) [reply spaces/blxpr.eth/blxpr.eth]: Done, thanks!

**Comment by gokaiorg** [comment]
Please rename your images from the example below `spaces/blxpr.eth/logo.png.png` to `spaces/blxpr.eth/logo.png`

**Comment by gokaiorg** [comment]
There is a conflict in your PR since `domains.json` file has been updated, can you resolve it?

**Comment by gokaiorg** [comment]
Closing this PR for now, feel free to open it again once you've done the changes.

**Pull request closed without merging**